src/_/CFG_GLPI.php

```diff
@@ -643,49 +553,49 @@
+    Rack::class               => "pics/impact/rack.png",
+    Software::class           => "pics/impact/software.png",
+    DatabaseInstance::class   => "pics/impact/databaseinstance.png",
 ];
 
 /**
  * All possible impact itemtypes: default + extra itemtypes that can be
  * added in GLPI configuration
  */
 $CFG_GLPI["impact_asset_types"] = $CFG_GLPI["default_impact_asset_types"] + [
-    AuthLDAP::getType()           => "pics/impact/authldap.png",
-    CartridgeItem::getType()      => "pics/impact/cartridgeitem.png",
-    Contract::getType()           => "pics/impact/contract.png",
-    CronTask::getType()           => "pics/impact/crontask.png",
-    DeviceSimcard::getType()      => "pics/impact/devicesimcard.png",
-    Entity::getType()             => "pics/impact/entity.png",
-    Group::getType()              => "pics/impact/group.png",
-    ITILCategory::getType()       => "pics/impact/itilcategory.png",
-    Line::getType()               => "pics/impact/line.png",
-    Location::getType()           => "pics/impact/location.png",
-    MailCollector::getType()      => "pics/impact/mailcollector.png",
-    Notification::getType()       => "pics/impact/notification.png",
-    Profile::getType()            => "pics/impact/profile.png",
-    Project::getType()            => "pics/impact/project.png",
-    Rack::getType()               => "pics/impact/rack.png",
-    SLM::getType()                => "pics/impact/slm.png",
-    SoftwareLicense::getType()    => "pics/impact/softwarelicense.png",
-    Supplier::getType()           => "pics/impact/supplier.png",
-    User::getType()               => "pics/impact/user.png",
-    Database::getType()           => "pics/impact/database.png",
+    AuthLDAP::class           => "pics/impact/authldap.png",
+    CartridgeItem::class      => "pics/impact/cartridgeitem.png",
+    Contract::class           => "pics/impact/contract.png",
+    CronTask::class           => "pics/impact/crontask.png",
+    DeviceSimcard::class      => "pics/impact/devicesimcard.png",
+    Entity::class             => "pics/impact/entity.png",
+    Group::class              => "pics/impact/group.png",
+    ITILCategory::class       => "pics/impact/itilcategory.png",
+    Line::class               => "pics/impact/line.png",
+    Location::class           => "pics/impact/location.png",
+    MailCollector::class      => "pics/impact/mailcollector.png",
+    Notification::class       => "pics/impact/notification.png",
+    Profile::class            => "pics/impact/profile.png",
+    Project::class            => "pics/impact/project.png",
+    Rack::class               => "pics/impact/rack.png",
+    SLM::class                => "pics/impact/slm.png",
+    SoftwareLicense::class    => "pics/impact/softwarelicense.png",
+    Supplier::class           => "pics/impact/supplier.png",
+    User::class               => "pics/impact/user.png",
+    Database::class           => "pics/impact/database.png",
 ];
```

Comment:
Using `::class` here result in the fact that all these classes are not loaded anymore during the GLPI bootstraping. I do not know if it improves the bootstraping duration (maybe a few milliseconds, but nor noticeable when opcache is already populated).

The changes made in tests were required due to this change.